docker-compose/scripts/docker_compose_run.sh

```
@@ -35,6 +35,16 @@ else
     cp_built_lock_file
 fi
 
+
+# Migrate the db, if needed.
+bundle exec rake db:migrate
+
+# Fix "inotify event queue has overflowed."
+# https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers
+sysctl fs.inotify.max_user_watches=524288
+sysctl fs.inotify.max_queued_events=524288
+sysctl fs.inotify.max_user_instances=524288
```

this crashed constantly when you made a bunch of JS changes, but I never knew why until @sadleb's earlier change to have Rails log to stdout. now that the error isn't being swallowed, let's fix it.

---

Whoa, interesting. So much trickiness with Listen. Remember the CPU running like a jet engine issue a few months back 🙃


~~Prob not worth doing since you already got it working this way, but I think ideally we would change Dockerfile to configure these in the image itself using something this:~~
```
RUN echo "fs.inotify.max_user_watches=524288 
fs.inotify.max_queued_events=524288
fs.inotify.max_user_instances=524288" >> /etc/sysctl.conf && \
sysctl -p
```
Welp, that doesn't work. I guess docker low-level kernel stuff is tricky. Lgtm!

---

Yeah I tried this a bunch of different ways and this was the only one that worked 🙃. Evidently you can't set persistent flags unless they're "[namespaced](https://docs.docker.com/engine/reference/commandline/run/#configure-namespaced-kernel-parameters-sysctls-at-runtime)", which fs.inotify isn't.